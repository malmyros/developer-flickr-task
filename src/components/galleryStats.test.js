import React from 'react';
import { shallow } from 'enzyme';
import GalleryStats from './galleryStats';

describe('gallery stats', () => {
  it('should render without crashing', () => {
    const component = shallow(<GalleryStats countTotal={10}
                                            countViews={2}
                                            countComments={3}/>);
    expect(component).toMatchSnapshot();
  });
});
