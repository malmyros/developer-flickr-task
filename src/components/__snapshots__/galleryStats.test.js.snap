// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`gallery stats should render without crashing 1`] = `
<ul
  className="gallery-stats"
>
  <li
    className="mb-1 text-muted"
  >
    10
     items
  </li>
  <li
    className="mb-1 text-muted"
  >
    2
     views
  </li>
  <li
    className="mb-1 text-muted"
  >
    3
     comments
  </li>
</ul>
`;
